radio/src/gui/128x64/radio_diaganas.cpp

```diff
@@ -22,12 +22,16 @@
 #include "opentx.h"
 #include "../../hal/adc_driver.h"
 
+#if defined(RADIO_BOXER)
+ #include "flysky_gimbal_driver.h"
+#endif
```

Comment:
Rather than lines like this one, and another example that I'll comment on, shouldn't this really be a `FLYSKY_GIMBALS` type define? Since  there is actually nothing RM Boxer specific about this code?

Reply:
Good point, addressed in latest commit 067cede.